api/src/utils/nova/addressHelper.ts

```diff
@@ -0,0 +1,146 @@
+/* eslint-disable import/no-unresolved */
+/* eslint-disable @typescript-eslint/no-redundant-type-constituents */
+/* eslint-disable @typescript-eslint/no-unsafe-argument */
+import {
+    Address,
+    AddressType,
+    AccountAddress,
+    Ed25519Address,
+    NftAddress,
+    AnchorAddress,
+    Utils,
+    ImplicitAccountCreationAddress,
+    RestrictedAddress,
+} from "@iota/sdk-nova";
+import { plainToInstance } from "class-transformer";
+import { IAddressDetails } from "../../models/api/nova/IAddressDetails";
+import { HexHelper } from "../hexHelper";
+
+export class AddressHelper {
```

Comment:
this whole thing is duplicated with the client one right? lets add a TODO and create a task so we try to improve this, we should have shared/common code we can reuse between both modules

Reply:
Created :point_right: : https://github.com/iotaledger/explorer/issues/1116